functions/_config.js

```diff
@@ -4,7 +4,9 @@ module.exports = {
   token: process.env.GITHUB_TOKEN,
   apiKey: process.env.MAILGUN_KEY,
   domain: process.env.MAILGUN_DOMAIN,
+  notionToken: process.env.NOTION_CONTEST_INTEGRATION_TOKEN,
+  notionDb: process.env.NOTION_CONTEST_DATABASE_ID,
```

Comment:
Trying to move away from this _config (#2137); a bit more explicit, maybe.

```suggestion
  notionContestDb: process.env.NOTION_CONTEST_DATABASE_ID,
```